src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2IntegrationTest.java

```diff
@@ -63,6 +66,49 @@ public void testDreamTumorNormal(final File tumorBam, final String tumorSample,
         Assert.assertTrue(fdr < 0.5);
     }
 
+    // make a pon with a tumor and then use this pon to call somatic variants on the same tumor
+    // if the pon is doing its job all calls should be filtered by this pon
+    @Test(dataProvider = "dreamSyntheticDataSample1")
+    public void testPon(final File tumorBam, final String tumorSample, final File normalBam, final String normalSample) throws Exception {
+        Utils.resetRandomGenerator();
+
+        final File ponVcf = createTempFile("pon", ".vcf");
+        final String[] createPonArgs = {
+                "-I", tumorBam.getAbsolutePath(),
+                "-tumor", tumorSample,
+                "-I", normalBam.getAbsolutePath(),
+                "-normal", normalSample,
+                "-R", b37_reference_20_21,
+                "-L", "20",
+                "-O", ponVcf.getAbsolutePath(),
+                "-artifact_detection_mode"
+        };
+
+        runCommandLine(createPonArgs);
+        new Main().instanceMain(makeCommandLineArgs(Arrays.asList("-F", ponVcf.getAbsolutePath()), "IndexFeatureFile"));
+
+        final File outputVcf = createTempFile("output", ".vcf");
+        final String[] callWithPonArgs = {
+                "-I", tumorBam.getAbsolutePath(),
+                "-tumor", tumorSample,
+                "-I", normalBam.getAbsolutePath(),
+                "-normal", normalSample,
+                "-normal_panel", ponVcf.getAbsolutePath(),
+                "-R", b37_reference_20_21,
+                "-L", "20",
+                "-O", outputVcf.getAbsolutePath()
+
+        };
+
+        runCommandLine(callWithPonArgs);
+
+        final Set<String> outputKeys = StreamSupport.stream(new FeatureDataSource<VariantContext>(outputVcf).spliterator(), false)
+                .filter(vc -> vc.getFilters().isEmpty())
+                .map(vc -> keyForVariant(vc)).collect(Collectors.toSet());
```

Comment:
since we expect outputKeys to be empty we could just `count()` after `.filter()` and check that it equals 0